### ISSUE TITLE
Runtime compiler is not loaded

### ISSUE THREAD

**Issue by Xepe** [devyumao/angular2-busy#55]
hello, 

Using the version 2.0.4 and angular cli, I am getting Uncaught (in promise): Error: Runtime compiler is not loaded. 

The error only rise when I try to use angular cli and the flag --prod

Regards

**Comment by hungshou** [comment]
me too

use angular2-busy 2.0.4 + aot + lazy loading , 

getting 

ERROR Error: Uncaught (in promise): Error: Cannot find module './setting/setting.module'.

ng build --> OK
ng build -prod --> Error
ng build -prod -aot=false --> OK

Regards

**Comment by shi1hh** [comment]
use ultimate/aot-loader 0.1.16 aot+lazy load
build >ok
but busy did not show

**Comment by Xepe** [comment]
I think that it is related to https://github.com/devyumao/angular2-busy/issues/60 also I created a fork with the solution

Regards.

**Comment by maplion** [comment]
I also experienced this issue.  I just yanked angular2-busy for the time being, but is there a good workaround?  @Xepe does your fork fix the issue?

**Comment by Xepe** [comment]
@maplion yes, you can use it https://www.npmjs.com/package/tixif-ngx-busy

**Comment by mavm84** [comment]
Same issue here, 
using this meanwhile:

`"angular2-busy": "https://github.com/dinusuresh/angular2-busy.git"`

https://github.com/dinusuresh/angular2-busy
